app/src/main/res/drawable/ic_loc_stop.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="19dp"
+    android:height="25dp"
+    android:viewportWidth="19"
+    android:viewportHeight="25">
+  <path
+      android:pathData="M0.2666,9.27C0.6,4.4006 4.5786,0.5873 9.3333,0.5873C14.088,0.5873 18.0666,4.4006 18.4,9.27C18.5168,11.5045 17.8333,13.7075 16.472,15.4833L9.976,24.1926C9.825,24.3951 9.5872,24.5144 9.3346,24.5144C9.082,24.5144 8.8443,24.3951 8.6933,24.1926L2.1946,15.4833C0.8333,13.7075 0.1497,11.5045 0.2666,9.27ZM5.7477,10.776C6.3521,12.2203 7.7677,13.1575 9.3333,13.15C11.4627,13.1426 13.1843,11.4127 13.1813,9.2833C13.1813,7.7177 12.2372,6.3066 10.79,5.7092C9.3428,5.1118 7.6783,5.4461 6.5739,6.5558C5.4695,7.6656 5.1433,9.3317 5.7477,10.776Z"
+      android:fillColor="#079DDC"
```

Comment:
should this and the other `fillColor`s refer to colors in `colors.xml`?

Reply:
That was Android's automatic import of svg files, those colors aren't really reused, so I think I'll stick with what the system created for now.